equipment_handler/src/adcs_handler.c

```diff
@@ -1358,29 +1372,49 @@ ADCS_returnState ADCS_save_orbit_params(void) {
 ADCS_returnState ADCS_get_current_state(adcs_state *data) {
     uint8_t telemetry[54];
     ADCS_returnState state;
+
     state = adcs_telemetry(ADCS_STATE, telemetry, 54);
     data->att_estimate_mode = telemetry[0] & 0xF;    // Refer to table 80
     data->att_ctrl_mode = (telemetry[0] >> 4) & 0xF; // Refer to table 78
     data->run_mode = telemetry[1] & 0x3;             // Refer to table 75
     data->ASGP4_mode = (telemetry[1] >> 2) & 0x3;    // Refer to table 87
+
+    // Offset: 12 (Table 149)
```

Comment:
Can we leave a block comment here explaining what happens in the next section - it's impossible to tell what is happening.

Reply:
Fixed in the most recent commit.